cedar-policy/src/api.rs

```diff
@@ -1705,7 +1707,17 @@ impl<'a> Diagnostic for ValidationWarning<'a> {
     }
 }
 
-/// unique identifier portion of the `EntityUid` type
+/// Unique identifier portion of the [`EntityUid`] type.
```

Comment:
```suggestion
/// Identifier portion of the [`EntityUid`] type.
```
Not necessarily required to be unique. (Although it would be confusing to have the same `EntityId` in multiple namespaces.)